vnext/fmt/packages.config

```diff
@@ -1,3 +0,0 @@
-﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
What happens if a user project consuming RNW, or an RNW community module has a packages.config file? Do we still restore these?